dtfs-central-api/src/v1/controllers/portal/facility/update-facility.controller.js

```diff
@@ -12,51 +13,56 @@ const withoutId = (obj) => {
   return cleanedObject;
 };
 
-const updateFacility = async (facilityId, facilityBody, dealId, user, routePath) => {
-  if (ObjectId.isValid(dealId) && ObjectId.isValid(facilityId)) {
-    const collection = await db.getCollection(MONGO_DB_COLLECTIONS.FACILITIES);
+const updateFacility = async (facilityId, facilityUpdate, dealId, user, routePath, auditDetails) => {
```

Comment:
Would prefer this to destructure params from an object

```suggestion
const updateFacility = async {(facilityId, facilityUpdate, dealId, user, routePath, auditDetails}) => {
```